fiftyone/utils/openimages.py

```diff
@@ -93,9 +95,20 @@ def __init__(
         seed=None,
         max_samples=None,
     ):
-        _label_types = _parse_label_types(label_types)
+
+        if label_types == None:
```

Comment:
Alternatively, it would be cleaner to refactor this logic into the `_parse_label_types` function.

You could still check for `points.csv` in this constructor as a heuristic for `version`, but then just pass `_parse_label_types(label_types, version)` where the rest of the logic is applied.

Reply:
I think the best approach here would be to check on disk what label types are available and parse the `version` automatically based on the presence of a `points.csv`. This way the `version` doesn't need to be provided explicitly, but all of the other methods that need the `version` will have it available (see my other comments).

It should also raise an informative error if a label type was specified that is not available.